sys/netbsd/sys.txt

```diff
@@ -190,6 +194,46 @@ ucred {
 	gid	gid
 }
 
+posix_spawn_file_actions {
+	size	int32
+	len	int32
+	fae	ptr[in, posix_spawn_file_actions_entry]
+}
+
+posix_spawn_file_actions_entry {
+	fae_action	int32
```

Comment:
Makes sense to describe this as flags with FAE_OPEN, FAE_DUP2, FAE_CLOSE values.